arbiter-core/src/tests/interaction.rs

```diff
@@ -253,3 +254,42 @@ async fn transaction_loop() -> Result<()> {
     }
     Ok(())
 }
+
+#[tokio::test]
+async fn test_pause_prevents_processing_transactions() {
```

Comment:
don't need to have `test_` in the name of a test, it's redundant.

Reply:
I know it's not necessary but it helps me and allows me to be precise when naming tests. If it bothers you we should chat more about it and maybe make changes to a style guide.